contract/src/start-game1-proposal.js

```diff
@@ -38,28 +44,25 @@ const publishBrandInfo = async (chainStorage, board, brand) => {
 
 /**
  * Core eval script to start contract
- *
- * @param {BootstrapPowers} permittedPowers
+ * XXX FIXME File '~/agoric-sdk/packages/vats/src/core/types.js' is not a module
```

Comment:
importing the ambient for side-effects should work.
https://github.com/Agoric/agoric-sdk/blob/a0d8229524d8a940c56d28309f67881557bc1ad3/packages/deploy-script-support/test/unitTests/test-coreProposalBehavior.js#L11-L14